__tests__/app.test.js

```diff
@@ -185,3 +187,102 @@ describe('GET /api/articles/:article_id/comments', () => {
       });
   });
 });
+
+describe('POST /api/articles/:article_id/comments', () => {
+  it('200: creates a new comment for a valid article id and responds with the new comment object', () => {
+    const testNewComment = {
+      username: 'butter_bridge',
+      body: 'This is a test comment body.',
+    };
+
+    return request(app)
+      .post('/api/articles/1/comments')
+      .send(testNewComment)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+
+        expect(comment).toMatchObject({
+          comment_id: expect.any(Number),
+          body: expect.any(String),
+          article_id: expect.any(Number),
+          author: expect.any(String),
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+        });
+      });
+  });
+
```

Comment:
I'd recommend adding another 201 for a test that ignores other additional properties that are passed 👍

Reply:
I've added a test for this and included `votes: 100, foo: 'bar'` in the request body with expectation for the following:

```javascript
expect(comment.votes).not.toBe(100);
expect(comment).not.toHaveProperty('foo');